modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceInventory.testcase

```diff
@@ -401,6 +512,237 @@ definition {
 		}
 	}
 
+	@description = "This is a test for COMMERCE-12326. If the SKU/UOM combination is set, the admin deletes a placed order, the inventory quantities from the SKU/UOM combination can be restocked correctly"
```

Comment:
If the SKU/UOM combination is set, when the admin deletes a placed order, can view the inventory quantities from the SKU/UOM combination be restocked correctly